index.js

```diff
@@ -121,6 +122,7 @@ async function createSentryConfig({ sentryOrg, sentryProject, sentryAuthToken })
   [defaults]
   project=${sentryProject}
   org=${sentryOrg}
+  pipeline=netlify/${packageJSON.version}
```

Comment:
Maybe something more specific, incase we have to build another plugin with netlify in the future. 
```netlify-build-plugin/${version}```